### ISSUE TITLE
withLocalize breaks navigationOptions for react-navigation

### ISSUE THREAD

**Issue by andyhbg** [ryandrewjohnson/react-localize-redux#122]
**Bug**

When I use **withLocalize** on a screen component together with react-navigation, **[static navigationOptions](https://reactnavigation.org/docs/en/headers.html)** is no longer called.

This works and the title is shown
```
class HomeScreen extends React.Component {
  static navigationOptions = {
    title: 'Home',
  };

  render() {
    return (
      <ScrollView>         
         <Button 
             onPress={() => null}
             title={this.props.translate('home.login')}
             <!-- Of course this only works when withLocalize is used -->
      </ScrollView>
    );
  }
}
export default HomeScreen;
```
but when I use `export default withLocalize(HomeScreen);` navigationOptions is not called and the title is not displayed. Everything else works well and the translations are shown. 

I have my own project but have also tried with a default expo tab template project and experience the same problem.

Maybe it's not that much help but you can see the app [here](https://expo.io/@andy_hbg/Localize). The Links screen is with withLocalize and the Settings screen without.

```
"dependencies": {
    "@expo/samples": "2.1.1",
    "expo": "29.0.0",
    "react": "16.3.1",
    "react-localize-redux": "^3.3.2",
    "react-native": "https://github.com/expo/react-native/archive/sdk-29.0.0.tar.gz",
    "react-navigation": "^2.9.3",
    "redux": "^4.0.0"
},
```

**Comment by thchia** [comment]
I will have a stab at this over the next few days.

@andyhbg I suspect it is related to [this](https://reactjs.org/docs/higher-order-components.html#static-methods-must-be-copied-over), if you are comfortable feel free to poke around first and see if you can confirm.

**Comment by andyhbg** [comment]
Hi @thchia!

Thanks for your feedback. I would love to help but I'm rather new to this so I'm not sure I feel that comfortable. I look around a bit and see what I can find.

For now, if I understand correct, react-redux connect uses [this](https://github.com/mridgway/hoist-non-react-statics) to solve the problem. Maybe that could be some help.

**Comment by thchia** [comment]
@andyhbg The library you linked to is the same one mentioned in the page I linked to from the React docs.

I'll probably won't be able to get to this for a few days, when I do I will update here.

**Comment by andyhbg** [comment]
Hmm, of course! Sorry about that and thank you for your help!

**Comment by andyhbg** [comment]
Proposed solution seems to solve the problem. It works at least in my projects and hopefully for others as well.

**Comment by ryandrewjohnson** [comment]
Fixed in #130 and published in `v3.5.0`.
